src/ui/zcl_abapgit_gui_page_diff.clas.abap

```diff
@@ -1111,6 +1111,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_DIFF IMPLEMENTATION.
       ro_html->add( '<th class="num">new</th>' ).           "#EC NOTEXT
       ro_html->add( '<th class="mark"></th>' ).             "#EC NOTEXT
       ro_html->add( '<th>code</th>' ).                      "#EC NOTEXT
+      ro_html->add( '<th class="collapse" onclick="onDiffCollapse(event);">▼</th>' ). "#EC NOTEXT
```

Comment:
Any idea how could I solve this abaplint issue?

**Contains non 7 bit ascii character
7bit_ascii**

Should I try to use an ASCII character ou replace it with an icon?

Reply:
suggest: add "zcl_abapgit_gui_page_diff" to exclude in abaplint.json, under the rule

Reply:
yea, moving it to the JS file might also work

Reply:
I added the exception. Thanks!

Reply:
Just FYI: there are unicode symbols escaping in html. This one would be `&#x25bc;` I guess

![image](https://user-images.githubusercontent.com/15635498/76536794-7be11d00-6485-11ea-8bc2-8f97445669f4.png)

But chevron icons look better anyway